vmms/dockerSSH.py

```diff
@@ -0,0 +1,323 @@
+#
+# dockerSSH.py - Implements the Tango VMMS interface to run Tango jobs in 
+#                docker containers. In this context, VMs are docker containers.
+#
+import random, subprocess, re, time, logging, threading, os, sys
+import config
+from tangoObjects import TangoMachine
+
+def timeout(command, time_out=1):
```

Comment:
It looks like we use the same code in all the VMMs. Can we make this a util method?